portal/static/js/main.js

```diff
@@ -2896,8 +2920,130 @@ var tnthDates = {
         }
         return toReturn
     },
-    "isDate": function(d) {
-        return Object.prototype.toString.call(d) === "[object Date]" && !isNaN( d.getTime() );
+    "isValidDefaultDateFormat": function(date, errorField) {
```

Comment:
Ivan, as discussed, I am hesitant to use Date.parse, as from what I read and experienced, it is returning inconsistent results across browsers.